### PR TITLE
Suggestions and Corrections.

### PR DESCRIPTION
**Pull request by Nikoleta-v3**
Following the corrections on the abstract and the citations that Vince did.

- `1f6b67a` A suggestion on the abstract. 
- `1703ca9` Because the first time noisy and probabilistic tournaments are mentioned you don't go through an explanation, I believe a reference is needed.
-  `ec4d376` A few typos and adding citations. Several strategies were not cited the first time they were
mentioned. I added the corresponding references. A few typos and mainly adding `.` in the figure/table captions.
- last commits are in sections `Methods` and `Discussion`. A few labels were fixed and a suggestion on a list order.

### CODE REVIEW

**Comment by Nikoleta-v3** [comment]
Please let me know if you disagree with anything and I make the changes.

File: main.tex [hunk 1]
```diff
@@ -45,7 +45,7 @@
     opponents, including many well-known and classic strategies. All
     the trained strategies win standard tournaments against the total collection
     of other opponents. The trained strategies and one particular human made
-    designed strategy are the top performers in noisy tournaments also.
+    designed strategy are the top performers in tournaments where also noise is introduced.
```
Comment (drvinceknight) [inline main.tex]: Not quite right grammatically: should be `where noise is also introduced` **but** I think this doesn't add much to what is already there so can you revert this.

File: main.tex [hunk 2]
```diff
@@ -488,12 +490,12 @@ \subsection{Standard Tournament}\label{sec:standard}
...
         \caption{Standard Tournament: number of wins per tournament (ranked by
         median over
-        \protect\input{./assets/standard_number_of_repetitions.tex}tournaments)}
+        \protect\input{./assets/standard_number_of_repetitions.tex}tournaments).}
         \label{fig:standard_winplot}
     \end{figure}
 \end{landscape}
 
-The number of wins of the top strategies of Table~\ref{tbl:standard_wins} are
+The number of wins of the top strategies of Table~\ref{tbl:standard_wins_top_winners} are
```
Comment (drvinceknight) [inline main.tex]: Good catch, there's an error here but your fix isn't right, it should be `tbl:standard_scores`

**Review by drvinceknight — CHANGES_REQUESTED** [review]
Thanks Nik, could you change a couple of things I've raised please.

**Comment by drvinceknight** [comment]
Thanks :+1:

**Pull request merged**